src/EditorFeatures/VisualBasicTest/UseNullPropagation/UseNullPropagationTests.vb

```diff
@@ -321,6 +321,90 @@ Class C
     Sub M(o As String)
         Dim x = [||]If (o Is Nothing, Nothing, o.Length)
     End Sub
+End Class")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseNullPropagation)>
+        Public Async Function TestWithNullableTypeAndReferenceEquals() As Task
+            Await TestInRegularAndScriptAsync(
+"
+Imports System
+
+Class C
+    Sub M(o As Object)
+        Dim v = [||]If (ReferenceEquals(o, Nothing), Nothing, o.ToString())
```

Comment:
so VB has 'ommitted' arguments which are funky and which may end up breaking some of the code htat assumes normal arguments.  Can you try this as a test:

```If (ReferenceEquals(o, ), Nothing, o.ToString())```
```If (ReferenceEquals(, nothing), Nothing, o.ToString())```

Note: i don't expect any actual code change, i just want to make sure we don't crash :)